setup.py

```diff
@@ -13,18 +20,43 @@ def get_description():
         return info + '\n\n' + changelog.read()
 
 
-with open(loc + '/requirements.txt') as f:
+with open(loc + '/requirements-micro.txt') as f:
     required = f.read().splitlines()
+if not micro:
+    with open(loc + '/requirements.txt') as f:
+        for line in f:
+            line = line.strip()
+            if line != '-r requirements-micro.txt':
+                required.append(line)
 
 with open('flask_monitoringdashboard/constants.json', 'r') as f:
     constants = json.load(f)
 
+if micro:
+    packages = setuptools.find_packages(include=[
+        'flask_monitoringdashboard',
+        'flask_monitoringdashboard.database',
+        'flask_monitoringdashboard.core',
+        'flask_monitoringdashboard.core.config',
+        'flask_monitoringdashboard.core.custom_graph',
+        'flask_monitoringdashboard.core.profiler',
+        'flask_monitoringdashboard.core.profiler.*',
+    ],
+        exclude=['flask_monitoringdashboard.controllers.*',
+                 'flask_monitoringdashboard.frontend.*',
+                 'flask_monitoringdashboard.views.*',
+                 'flask_monitoringdashboard.static.*',
+                 'tests', 'test.*']
```

Comment:
we should also exclude the tests in non-micro mode

Reply:
For whatever reason find_packages has been giving me pain lately. I had thought you were excluding tests via specifying the test suite later - but maybe not? 

Because of the explicit includes up above these are technically not needed and likely just got copied from wherever the last time i used exclude was. Are you able to double check that it currently includes them on your end? Something in the way I am building could be funky.